src/pages/Models/Details/_model-details.scss

```diff
@@ -16,6 +16,11 @@
     @media (min-width: $breakpoint-m-large) {
       grid-template-columns: 350px 1fr;
     }
+
+    &[aria-disabled="true"] {
+      opacity: 0.5;
```

Comment:
I think there is a 'disabled' state in Vanilla.

Reply:
Not as a standalone util.